### ISSUE TITLE
it seem the url parameter can be removed?

### ISSUE THREAD

**Issue by TiaNex-Com** [yiisoft/demo#573]
[CommonViewInjection.php](https://github.com/yiisoft/demo/blob/master/blog/src/ViewInjection/CommonViewInjection.php)
there is $urlGenerator in view now,

### Additional info

| Q                | A
| ---------------- | ---
| Version          | 1.0.?
| PHP version      | 
| Operating system |

**Comment by vjik** [comment]
Yes. `url` here duplicate `urlGenerator` from [params](https://github.com/yiisoft/demo/blob/20e27abd37d3d31838bd7dab80be1ab65812d9c6/blog/config/common/params.php#LL85C14-L85C26).

**Comment by samdark** [comment]
It is now there but we should rename it from `url` to `urlGenerator`. Reasons:

1. It's less likely to cause naming conflict.
2. It's like there in yiisoft/app already.
